first.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+// 1 ---------------
+
+const numberSetOne = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+function modifyNumbers(numbers) {
+    const modifyNumberSet = [];
+
+    for (let i = 0; i < numbers.length; i++) {
+        // if (numbers[i] % 2 === 0) {
+        //     modifyNumberSet.push(numbers[i] += 1);
+        // } else {
+        //     modifyNumberSet.push(numbers[i] -= 1);
+        // }
+        let currentNumber = numbers[i];
+        modifyNumberSet.push(currentNumber % 2 === 0 ? currentNumber += 1 : currentNumber -= 1);
+
+    }
+    return modifyNumberSet;
+}
+
+console.log(modifyNumbers(numberSetOne));
+
+
+// 2 ---------------
+
+function capitalizeWords(words) {
+    return words.map(word => {
+
+        let firstLetter = word.charAt(0);
+        firstLetter = firstLetter.toUpperCase();
+
+        let remainingLetters = word.slice(1);
+        remainingLetters = remainingLetters.toLowerCase();
+
+        let capitalizedWord = firstLetter + remainingLetters;
+
+        return capitalizedWord;
+    });
+}
```

Comment:
```javascript
const firstLetter = word[0].toUpperCase();
const restLetters = word.slice(1).toLowerCase()

return `${firstLetter}${restLetters}` 
``` 
For more details check this out: https://learn.javascript.ru/types#stroka
looks shorter and mach cleaner